bin/install-node-nvm.sh

```diff
@@ -72,6 +72,21 @@ npm install
 # Make sure npm is up-to-date
 npm install npm -g
 
+# Check if yarn is installed
+if [ "$TRAVIS" != "true" ] && ! command_exists "yarn"; then
```

Comment:
Curious why `yarn` is used for mobile? Can't we just make it "optional" and rely on npm consistently?